chat_shell/chat_shell/history/loader.py

```
@@ -637,20 +637,161 @@ def _build_document_text_prefix(
+        logger.info(
+            f"[_fetch_kb_documents_content] Fetched {len(documents)} docs "
+            f"({len(content)} chars) from KB {knowledge_base_id}"
+        )
+        return content
+
+    except ImportError as e:
+        logger.warning(
+            f"[_fetch_kb_documents_content] Import error (HTTP mode?): {e}"
+        )
+        return ""
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)